### PR TITLE
Setup package structure using kwb.pkgbuild

### PR DESCRIPTION
**Pull request by hsonne**
The script that was used to setup this
package structure is in
inst/extdata/init_package.R.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -34,3 +34,4 @@ vignettes/*.pdf
 
 # Shiny token, see https://shiny.rstudio.com/articles/shinyapps.html
 rsconnect/
```
Comment (fabianmoronzirfas) [inline .gitignore]: I think the .gitignore needs some more conntent

```plain
 # History files
 .Rhistory
 .Rapp.history

 # Session Data files
 .RData

 # Example code in package build process
 *-Ex.R

 # Output files from R CMD build
 /*.tar.gz

 # Output files from R CMD check
 /*.Rcheck/

 # RStudio files
 .Rproj.user/

 # produced vignettes
 vignettes/*.html
 vignettes/*.pdf

 # OAuth2 token, see https://github.com/hadley/httr/releases/tag/v0.3
 .httr-oauth

 # knitr and R markdown default cache directories
 /*_cache/
 /cache/

 # Temporary files created by R markdown
 *.utf8.md
 *.knit.md

 # Shiny token, see https://shiny.rstudio.com/articles/shinyapps.html
 rsconnect/

 # History files
 .Rhistory
 .Rapp.history

 # Session Data files
 .RData

 # Example code in package build process
 *-Ex.R

 # Output files from R CMD build
 /*.tar.gz

 # Output files from R CMD check
 /*.Rcheck/

 # RStudio files
 .Rproj.user/

 # produced vignettes
 vignettes/*.html
 vignettes/*.pdf

 # OAuth2 token, see https://github.com/hadley/httr/releases/tag/v0.3
 .httr-oauth

 # knitr and R markdown default cache directories
 /*_cache/
 /cache/

 # Temporary files created by R markdown
 *.utf8.md
 *.knit.md
```

File: .gitlab-ci.yml [hunk 2]
```diff
@@ -0,0 +1,24 @@
+##############################################################################
```
Comment (fabianmoronzirfas) [inline .gitlab-ci.yml]: We don't need this file

File: .here [hunk 3]
```diff
@@ -0,0 +1 @@
+
```
Comment (fabianmoronzirfas) [inline .here]: What is this file for?

File: .travis.yml [hunk 4]
```diff
@@ -0,0 +1,13 @@
+language: r
+sudo: required
```
Comment (fabianmoronzirfas) [inline .travis.yml]: Shouldn't we also define a fixed R version? I think it is like this:

```yml
env:
   - TRAVIS_R_VERSION=3.5.2
```

File: LICENSE.md [hunk 5]
```diff
@@ -0,0 +1,21 @@
+# MIT License
+
+Copyright (c) 2019 Kompetenzzentrum Wasser Berlin gGmbH (KWB)
```
Comment (fabianmoronzirfas) [inline LICENSE.md]: I guess one license file is enough. No need for a second one in [M↓]

File: NEWS.md [hunk 6]
```diff
@@ -0,0 +1,7 @@
+# fhpredict 0.0.0.9000
+
+* Added a `NEWS.md` file to track changes to the package.
```
Comment (fabianmoronzirfas) [inline NEWS.md]: IMHO should be `changelog.md` not NEWS.md

Comment (fabianmoronzirfas) [reply NEWS.md]: Can't say anything about tidyverse. If it is like this in R package land pls ignore

File: README.md [hunk 7]
```diff
@@ -1,2 +1,24 @@
...
+
+This R package provides all functions that are
+required for the Web application that is developed within the
+Flusshygiene project.
+
+## Installation
+
+```r
+#install.packages("remotes", repos = "https://cloud.r-project.org")
+remotes::install_github("KWB-R/fhpredict")
```
Comment (fabianmoronzirfas) [inline README.md]: still points to KWB but you already mentioned that in #5

File: _pkgdown.yml [hunk 8]
```diff
@@ -0,0 +1,13 @@
+authors:
```
Comment (fabianmoronzirfas) [inline _pkgdown.yml]: What is this file for?

File: index.Rmd [hunk 9]
```diff
@@ -0,0 +1,16 @@
+[![Appveyor build Status](https://ci.appveyor.com/api/projects/status/github/KWB-R/fhpredict?branch=master&svg=true)](https://ci.appveyor.com/project/KWB-R/fhpredict/branch/master)
```
Comment (fabianmoronzirfas) [inline index.Rmd]: Duplicate content. Same as README? Or is this a R-package-land thing?

File: .gitlab-ci.yml (hunk 2 above)
Comment (hsonne) [reply .gitlab-ci.yml]: I agree.

File: .here (hunk 3 above)
Comment (hsonne) [reply .here]: I don't know. Need to ask my colleague Michael Rustler (@mrustl) who is available next week.

File: NEWS.md (hunk 6 above)
Comment (hsonne) [reply NEWS.md]: This file is required (or at least recognised) by the R packaging mechanism and will show up in the help browser.

File: _pkgdown.yml (hunk 8 above)
Comment (hsonne) [reply _pkgdown.yml]: I think it is for automating package documentation.

File: .here (hunk 3 above)
Comment (fabianmoronzirfas) [reply .here]: okay. For now we just keep it here. If we don't need it we do some housekeeping later on

**Comment by fabianmoronzirfas** [comment]
This PR is outdated and superseded by #6

**Pull request closed without merging**